CMakeLists.txt

```diff
@@ -223,6 +223,7 @@ if ("${CMAKE_SYSTEM_NAME}" MATCHES "(Free|Net|Open|DragonFly)BSD")
     include_directories(${YAML_CPP_INCLUDE_DIR})
 else()
     find_package(yaml-cpp REQUIRED HINTS /usr/lib32/cmake/yaml-cpp)
+    include_directories(${YAML_CPP_INCLUDE_DIR})
```

Comment:
The two branches are nearly identical and maybe hints are used only as hints, in addition to regular paths - so the two branches can be probably merged

Reply:
Yeah, I think we can just remove the conditional branching entirely. See also the original discussion here: https://github.com/OpenLoco/OpenLoco/pull/385/files#r392202834